toolControl:
  topic: '/mixed_platform/platform_state'
  toolPos: 
    p: [0.0,0.0,0.0,5000.0,0.0]
    d: [10.0,10.0,100.0,100.0,100.0]
  toolSpeed:
    p: [2500.0,2500.0,5000.0,5000.0,2500.0]
    d: [10.0,10.0,100.0,100.0,100.0]